### PR TITLE
Update Witsoftware Description and Offices description

### PR DESCRIPTION
**Pull request by ClaymorePT**
This commit updates the WitSoftware description and offices purpose.
Not all WitSoftware offices exist for development purposes

Signed-off-by: Carlos Ferreira <carlosmf.pt@gmail.com>

### CODE REVIEW

**Comment by vpereira01** [comment]
The description should be very short, check https://github.com/marmelo/tech-companies-in-portugal/blob/master/CONTRIBUTING.md

Furthermore I think only development offices should be listed as this seems a listing for devs and not for other functions.

**Comment by ClaymorePT** [comment]
@vpereira01 Please review once again!

**Comment by ClaymorePT** [comment]
@vpereira01 Please review the contribution once again.

**Comment by vpereira01** [comment]
Seems OK now (I don't don't the repo, I'm just trying to help)

**Comment by dfilipeloja** [comment]
@ClaymorePT seems that "Lisboa" is missing.

File: README.md [hunk 1]
```diff
@@ -217,5 +217,5 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | [Celfocus](https://www.celfocus.com/) [:rocket:](https://www.celfocus.com/home/who-we-are/join-us) | Vodafone technology development.  | `Lisboa` `Porto` |
 | [GoContact](https://www.gocontact.pt/) [:rocket:](https://www.gocontact.pt/recrutamento/) | Company specialized in integrated solutions for Contact Centers. | `Aveiro` `Lisboa`<br>`Porto`|
 | [Talkdesk](https://www.talkdesk.com) [:rocket:](https://www.talkdesk.com/careers/) | Enterprise cloud contact center. | `Coimbra` `Lisboa`<br>`Porto` |
-| [WIT Software](https://www.wit-software.com) [:rocket:](https://www.wit-software.com/careers/) | Mobile, Micro-Payment, OTT Media, RCS (Rich Communication Services), Communication Gateways and other related software development for telecommunications services.  | Development Offices: `Coimbra` `Leiria` `Porto` `Aveiro` <br>Sales Offices: `Lisboa` `London` |
+| [WIT Software](https://www.wit-software.com) [:rocket:](https://www.wit-software.com/careers/) | Software development for telcom services (OTT, RCS, Mobile, Web & others). | `Coimbra` `Leiria` <br>`Porto` `Aveiro` |
```
[the remark this answers is not in the document]
Comment (ClaymorePT) [reply README.md]: WitSoftware does not hire developers for their office in Lisbon. It's a Sales office, not a development office.

[the remark this answers is not in the document]
Comment (ClaymorePT) [reply README.md]: The office's are sorted in the order in which they were opened. First Coimbra, then Leiria, Porto and the most recent one was in Ilhavo, Aveiro.

**Comment by afonsojramos** [comment]
@ClaymorePT Lisboa remains a Wit Software Office and thus, should remain. 

Offices must be ordered alphabetically an not by opened-date.

**Comment by ClaymorePT** [comment]
@afonsojramos Then what about their London Office?

**Comment by afonsojramos** [comment]
@ClaymorePT no since the list is for Portugal only. If not, you can imagine that many would have tens of offices on the list...

**Comment by ClaymorePT** [comment]
@afonsojramos Fixed alphabetic order and re-added Lisbon Office.

**Comment by afonsojramos** [comment]
@marmelo seems good to merge

**Comment by marmelo** [comment]
I believe the description is still too big which results in a broken layout.
Nevertheless, I am merging this as is because there are other companies with similar issues.

**Pull request merged**